core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/TurtleParser.java

```diff
@@ -278,12 +278,11 @@ protected void parsePrefixID() throws IOException, RDFParseException, RDFHandler
 		skipWSC();
 
 		// Read the namespace URI
-		IRI namespace = parseURI();
+		String namespaceStr = parseURI().toString();
 
-		// Store and report this namespace mapping
 		String prefixStr = prefixID.toString();
-		String namespaceStr = namespace.toString();
 
+		// Store and report this namespace mapping
```

Comment:
This is just to clean up some code that i stumbled upon while fixing a bug else where.